### PR TITLE
Add new commands, delegate get voters and votes - Closes #137

### LINKED ISSUE #137: Add get votes and get voters commands
Related issue: 
LiskHQ/lisk-commander/issues/168
LiskHQ/lisk-commander/issues/643

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #137

### CODE REVIEW

File: user-guide/commands/commands.md [hunk 1]
```diff
@@ -351,6 +353,68 @@ EXAMPLES
...
+  delegate:voters lightcurve,4miners.net --limit 20 --offset 5 --sort publicKey:asc --pretty
+```
+
+## Get votes of a delegate
+
+Gets votes information for given delegate(s) from the blockchain.
+
+```
+USAGE
+  $ lisk delegate:votes USERNAMES
```
Comment (mitsuaki-u) [inline user-guide/commands/commands.md]: `USERNAMES` -> `ADDRESSES`

File: user-guide/commands/commands.md [hunk 2]
```diff
@@ -351,6 +353,68 @@ EXAMPLES
...
+
+ARGUMENTS
+  USERNAMES  Comma-separated username(s) to get information about.
```
Comment (mitsuaki-u) [inline user-guide/commands/commands.md]: same as above

File: user-guide/commands/commands.md [hunk 3]
```diff
@@ -351,6 +353,68 @@ EXAMPLES
...
+
+  --[no-]pretty    Prints JSON in pretty format rather than condensed. Has no effect if the output is set to table. You
+                   can change the default behaviour in your config.json file.
+
+
+DESCRIPTION
+  Gets voters information for given delegate(s) from the blockchain.
+
+EXAMPLES
+  delegate:votes lightcurve
```
Comment (mitsuaki-u) [inline user-guide/commands/commands.md]: update examples to use addresses

**Review by mitsuaki-u — CHANGES_REQUESTED** [review]

File: user-guide/commands/commands.md (hunk 1 above)
Comment (Tschakki) [reply user-guide/commands/commands.md]: Are Usernames working as well or only adresses?

Comment (mitsuaki-u) [reply user-guide/commands/commands.md]: only addresses for votes. for voters username works.

**Review by mitsuaki-u — DISMISSED** [review]

**Review by mitsuaki-u — APPROVED** [review]

**Pull request merged**